packages/auth0-fastify-api/src/index.ts

```diff
@@ -65,25 +65,41 @@ async function auth0FastifApi(fastify: FastifyInstance, options: Auth0FastifyApi
     customFetch: options.customFetch,
   });
 
-  const replyWithError = (reply: FastifyReply, statusCode: number, error: string, errorDescription: string) => {
-    return reply
-      .code(statusCode)
-      .header(
-        'WWW-Authenticate',
-        `Bearer error="${error.replaceAll('"', '\\"')}", error_description="${errorDescription.replaceAll('"', '\\"')}"`
-      )
-      .send({
-        error: error,
-        error_description: errorDescription,
-      });
+  const replyWithError = (
+    reply: FastifyReply,
+    statusCode: number,
+    error: string,
+    errorDescription: string,
+    ommitHeaderDetails = false
+  ) => {
+    const headerValue = ommitHeaderDetails
+      ? `Bearer`
```

Comment:
We actually have to have an auth-param component here in order for the www-authenticate to be valid as per its definition.

This can the form of `WWW-Authenticate: Bearer realm="${currentUrlOrSomeArbitraryIdentifier}"`